packages/openchs-android/src/App.js

```
@@ -92,13 +94,61 @@ class App extends Component {
+        
+        const { type, id, entityType, rawParams } = parsedLink;
+        
+        // Navigate based on deep link type
+        switch (type) {
+            case 'enrollment':
+            case 'encounter':
+            case 'registration':
+                // Navigate to the appropriate form
+                // The actual navigation will be handled by the current view
+                // Store the deep link info in global context for the view to use
+                const globalContext = GlobalContext.getInstance();
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)